### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.1.1 - make carousel keyboard operable

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1284]
# Feature Request

## Description
carousel controls are not currently keyboard operable;

## Solution
show links in slides and controls on keyboard focus; 
convert spans in status bar into buttons or give them role="button"
![image](https://github.com/benjaminkott/bootstrap_package/assets/132338236/ffa56011-997c-4cb0-a5e8-ae9af22a97b3)
give button corresponding to current slide aria-pressed="true"

## Note

The carousel has a number of other issues, such as poor colour contrast, a flaw in html semantics, a missing aria-role and no stop/pause button. Recommendation: rebuild carousel according to W3C tutorial https://www.w3.org/WAI/tutorials/carousels/ or https://tollwerk.de/projekte/tipps-techniken-inklusiv-barrierefrei/barrierefreie-karussells

**Comment by SarahEdding** [comment]
QA done, all fixed
